### ISSUE TITLE
Uldingerongel

### ISSUE THREAD

**Issue by johnbent** [tekinged/missing#479]
Too much. 

Aika el ildisel a kall a kuk kmal uldingerongel.

**Comment by Lukes-Imeyuns** [comment]
Yep!

Excessive.

**Comment by Gnalu** [comment]
Yes or excessive. Aika el omelkingem el mei er ngak a ko er a kma el
uldingerongel a delcherul. Your responses to me is a bit excessive and
little out of content.
